src/pathology-report/ReportComponent.resource.ts

```diff
@@ -1,28 +1,75 @@
 import { openmrsFetch } from '@openmrs/esm-framework';
 
+const encounterTypeUuid = '4d911f5c-26fe-102b-80cb-0017a47871b2';
+const formName = 'CURRENT PROGRAMME ADULTE VIH - RENDEZ-VOUS - Rwanda 2.1';
+const sampleStatusUuid = '3ce163d8-26fe-102b-80cb-0017a47871b2';
+const referralStatusUuuid = '3cd93302-26fe-102b-80cb-0017a47871b2';
+const sampleDropOffUuid = '13662f27-9be5-4595-8bab-07b0e859d9f4';
+const phoneNumberAttrUuid = 'd6bcc287-4576-4264-961b-6bf1c08fbf68';
+const healthCenterAttrTypeUuid = '8d87236c-c2cc-11de-8d13-0010c6dffd0f';
+const rwinkwavuHospitalUuid = '38c4b94b-97a6-4f2c-bca6-c28a37cf60ed';
+const rwinkwavuHospitalName = 'Rwinkwavu Hospital';
+const rwinkwavuHealthcenterName = 'Rwinkwavu Health Center';
```

Comment:
It's a matter of preference, but I'm a fan of `ALL_CAPS` for things that are very formally constants like these.